eng/Versions.props

```diff
@@ -305,6 +305,12 @@
     <UsingToolMicrosoftNetCompilers Condition="'$(BootstrapBuildPath)' == ''">true</UsingToolMicrosoftNetCompilers>
     <UseVSTestRunner>true</UseVSTestRunner>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(NoPdbConversion)' == 'true'">
```

Comment:
>NoPdbConversion [](http://example.com/codeflow?start=31&length=15)

The extra property is unnecessary. Setting `/p:UsingToolPdbConverter=false` on msbuild command line overrides all values set in targets files.